### PR TITLE
ROC-4470: AKS setup

### PR DESCRIPTION
**Pull request by markry-kainos**
Resolves # . (This is applicable only if this pull request relates to a GitHub issue, delete the line otherwise)

Notes:
*
*
*

### CODE REVIEW

File: Jenkinsfile_parameterized [hunk 1]
```diff
@@ -10,9 +10,16 @@ properties([
 ])
 
 List<LinkedHashMap<String, Object>> secrets = [
-  secret('smoke-test-solicitor-username', 'SMOKE_TEST_SOLICITOR_USERNAME'),
+  secret('smoke-test-citizen-username', 'SMOKE_TEST_CITIZEN_USERNAME'),
```
Comment (timja) [inline Jenkinsfile_parameterized]: solicitor me thinks?

File: src/kubernetes/README.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+https://kubernetes.io/docs/reference/kubectl/cheatsheet/
+
+```
+kubectl config view                         # Show Merged kubeconfig settings.
+kubectl config current-context              # Display the current-context
+```
+
+## Known Issues
+
+- Emails will include wrong domain - AAT claim-store is used as backend.
```
Comment (timja) [inline src/kubernetes/README.md]: not sure if legal sends any emails with a link

File: Jenkinsfile_parameterized (hunk 1 above)
Comment (markry-kainos) [reply Jenkinsfile_parameterized]: hehe - i just spotted!  :))) changing now 👍

File: src/kubernetes/README.md (hunk 2 above)
Comment (markry-kainos) [reply src/kubernetes/README.md]: i might actually just link to citizen frontend readme - what you think? both are CMC...

Comment (timja) [reply src/kubernetes/README.md]: Sounds good, no point duplicating

**Comment by markry-kainos** [comment]
@timja morning :) this lgtm now can you review and hopefully approve pls - green build in jenkins 👍 https://build.platform.hmcts.net/job/HMCTS_CMC/job/cmc-legal-rep-frontend/view/change-requests/job/PR-385/

**Pull request merged**